newjersey/d4ad
# Scott/1425 feature flags

Resolves [NJWE-1425](https://fearless.jira.com/browse/NJWE-1425)

- This combines `master` and `v2.1` branch and should ultimately remove the need to have `v2.1` 
- keeps all features that aren't ready for production behind feature flags.

-----

~~@ChelseaKR I am just now realizing that I merged 2.1 into master rather than the other way around, there might be some extra stuff changed that I didn't intend to change. Please let me know if something seems wrong, while I look into this.~~

-----
### In order to test:
- add these variables into your fronted `.env` file
```
REACT_APP_FEATURE_CAREER_PATHWAYS=false
REACT_APP_FEATURE_MULTILANG=false
REACT_APP_FEATURE_PINPOINT=false
```
- toggle them on and off and restart the frontend project to see them get added or removed
